### DISCUSSION TITLE
Feature Request: onScroll callback for infinite scrolling and pulling data from the backend

### DISCUSSION THREAD

**Discussion by vozmarkov** [nick-keller/react-datasheet-grid#269] [Feature requests]
I am trying to implement infinite scrolling and pulling data with prisma / trpc 10 using useInfiniteQuery
but I have no way of detecting where the scroll bar is, I tried the ref, but it doesnt return ref of a div container where the scroll Y is only DataSheetGridRef.

**Comment by nick-keller** [comment]
Sounds faire, probably in next release

**Reply by vozmarkov** [reply]
When is the next release approximately?

**Reply by l0g1x** [reply]
I am in need of infinite scroll as well!

**Reply by nick-keller** [reply]
I don't have the bandwidth at the moment, but feel free to make a PR if you feel like it!

**Reply by hussamkhatib** [reply]
someone opened a PR for this :tada:  https://github.com/nick-keller/react-datasheet-grid/pull/290
